app/src/main/java/com/mathewsachin/fategrandautomata/util/SwipeLocations.kt

```diff
@@ -0,0 +1,27 @@
+package com.mathewsachin.fategrandautomata.util
+
+import android.os.Build
+import com.mathewsachin.fategrandautomata.scripts.ISwipeLocations
+import com.mathewsachin.libautomata.Location
+import javax.inject.Inject
+
+/**
+ * Android 8 added support for continued gestures, so we can do precise swiping.
+ * On Android 7, long swipes cause weird behaviour, so we need different locations.
+ */
+class SwipeLocations @Inject constructor() : ISwipeLocations {
```

Comment:
Did you come up with these values by trial and error?

Reply:
pretty much trial and error

Reply:
other than the values for Android 7 which were already in the app